itou/www/apply/tests/tests_process.py

```diff
@@ -971,6 +974,25 @@ def test_eligibility_for_siae_not_subject_to_eligibility_rules(self, *args, **kw
         response = self.client.get(url)
         assert response.status_code == 404
 
+    def test_eligibility_for_siae_with_suspension_sanction(self, *args, **kwargs):
+        """Test eligibility for an Siae that has been suspended."""
+
+        job_application = JobApplicationSentByPrescriberOrganizationFactory(
+            state=JobApplicationWorkflow.STATE_PROCESSING,
+            job_seeker=JobSeekerWithAddressFactory(),
+        )
+        Sanctions.objects.create(
+            evaluated_siae=EvaluatedSiaeFactory(siae=job_application.to_siae),
+            suspension_dates=InclusiveDateRange(timezone.localdate()),
```

Comment:
_nit_ : Pour "indiquer" que l'on veux juste être sanctionné et que la date importe peu ? :shrug: 
```suggestion
            suspension_dates=InclusiveDateRange(),
```

Reply:
L'objet Sanction peut contenir différentes sanctions (formation obligatoire, suspension, etc) et peut donc ne pas contenir de suspension.

Reply:
Mais dans ce cas là `suspension_dates` est à `None`, non ? Car de ce que j'ai compris de `InclusiveDateRange()` et `DateRange()` c'est que si la borne n'est pas spécifiée alors ça utilise, plus ou moins, `infinity` donc `InclusiveDateRange()` contiendrais toutes les dates possibles et ne serais donc pas _falsy_.

Mais j'avoue qu'en y repensant `InclusiveDateRange()` laisse plutôt supposer que c'est "vide" alors qu'en fait c'est "plein" :sweat_smile:.

Reply:
Oui, je pense effectivement que `InclusiveDateRange()` marcherait mais je préfère rester sur des tests crédibles :stuck_out_tongue: